### ISSUE TITLE
add ${workspaceFolder}  to "PYTHONPATH" in launch.json by default

### ISSUE THREAD

**Issue by Akopov4** [microsoft/vscode-python-debugger#343]
Hi, I'm new User of VScode. Previously I used Jetbrains Pycharm and not for a long time ago switched to VSCode.
Please add 
"env": {
                "PYTHONPATH": "${workspaceFolder}"
            }
as default configuration parameters, when create to launch.json
Pycharm did it for a long time ago
Thank you in advance

**Comment by shtratos** [comment]
I support this feature request, in general I think we can do better with out-of-the-box experience - I was surprised how much effort and setup it takes to just start debugging any Python code in VSCode for a new project as opposed to PyCharm where it "just works".

**Comment by abeschornerVF** [comment]
Plus one. So we can finally get rid of "source"ing env.sh each and every time.

**Comment by xingzhisg** [comment]
No, please DO NOT make this as default. For those who use [`src` layout](https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout/), this is exactly the settings that we want to avoid.

For any new project, just create a `.env` file in the project's root directory by assigning the environment variable `PYTHONPATH=<absolute path to this project root>` will solve the problem for this requirement.

**Comment by mikeweltevrede** [comment]
> No, please DO NOT make this as default. For those who use [`src` layout](https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout/), this is exactly the settings that we want to avoid.
> 
> For any new project, just create a `.env` file in the project's root directory by assigning the environment variable `PYTHONPATH=<absolute path to this project root>` will solve the problem for this requirement.

I think this is a tad problematic too: it requires every developer on each project to manually create this file (people will clone the repo in different locations, unless you want to enforce the same path for everyone).

**Comment by morning-start** [comment]
Maybe it is also possible to add "envFile": "${workspaceFolder}/.env" by default in the launch.json. Of course, it is best to be able to read the result in the Python extension.

![Image](https://github.com/user-attachments/assets/7b848e0a-20d8-44ba-a4f9-54842755782b)
